src/LiveDevelopment/LiveDevMultiBrowser.js

```diff
@@ -241,7 +241,7 @@ define(function (require, exports, module) {
      * @return {string}
      */
     function _resolveUrl(path) {
-        return _server && _server.pathToUrl(path);
+        return _server && path;
```

Comment:
The only other place I see this being used is in [LiveDevMultiBrowser] (https://github.com/humphd/brackets/blob/bramble/src/LiveDevelopment/LiveDevMultiBrowser.js#L552) when the "ConnectionConnect" event is triggered. However, our code doesn't enter this part because our Transport doesn't trigger this event.